### PR TITLE
Playground redesign

### PR DESCRIPTION
**Pull request by XVincentX**

### CODE REVIEW

File: playground/components/Playground.js [hunk 1]
```diff
@@ -35,21 +35,34 @@ class Playground extends React.Component {
 
   render() {
     return (
-      <div className="playgrund-app">
-        <div className="column">
-          <EditorComponent errors={this.state.parseResult.errors} />
+      <div className="playground-app">
```
Comment (Baggz) [inline playground/components/Playground.js]: _**Nit:**_ camelCase please.

File: playground/styles/layout.styl [hunk 2]
```diff
@@ -21,3 +21,50 @@
...
+.ico
+  &:before
+    content ''
+    display block
+    width 17px
+    height 17px
+    float left
+    margin-right 6px
+
+.githubIco
```
Comment (Baggz) [inline playground/styles/layout.styl]: _**Nit:**_ It's GitHub, so please use `gitHub`.

File: playground/styles/layout.styl [hunk 3]
```diff
@@ -21,3 +21,50 @@
...
+  &:before
+    background: url('icon-github.svg') no-repeat
+
+.documentationIco
+  &:before
+    background: url('icon-documentation.svg') no-repeat
```
Comment (Baggz) [inline playground/styles/layout.styl]: _**Nit:**_ Please use camelCase to be consistent with the rest of the codebase.

File: playground/styles/layout.styl [hunk 4]
```diff
@@ -21,3 +21,50 @@
...
+    content ''
+    display block
+    width 17px
+    height 17px
+    float left
+    margin-right 6px
+
+.githubIco
+  &:before
+    background: url('icon-github.svg') no-repeat
```
Comment (Baggz) [inline playground/styles/layout.styl]: _**Nit:**_ Please use camelCase to be consistent with the rest of the codebase.

File: playground/components/Playground.js [hunk 5]
```diff
@@ -35,21 +35,34 @@ class Playground extends React.Component {
...
-          <EditorComponent errors={this.state.parseResult.errors} />
+      <div>
+        <div className="header">
+        <div className="leftSide"><span>Attributes Kit Playground</span></div>
+        <div className="rightSide">
+          <span>
+            <a target="_blank" className="gitHubIco" href="https://github.com/apiaryio/attributes-kit">Attributes Kit Repository</a>
+          </span>
+          <span>
+            <a target="_blank" className="documentationIco" href="https://github.com/apiaryio/mson">Mson Documentation</a>
```
Comment (kylef) [inline playground/components/Playground.js]: MSON*

Comment (XVincentX) [inline playground/components/Playground.js]: Ok!
Keep in mind it has got “In Progress” label, so review it does not make much sense.

**Comment by XVincentX** [comment]
:(

**Pull request closed without merging**